### PR TITLE
Locales (it_IT): fix typo and remove periods

### PR DESCRIPTION
**Pull request by SuperSamus**

### CODE REVIEW

File: locales/it_IT/pcsx2_Main.po [hunk 1]
```diff
@@ -5,7 +5,7 @@ msgstr ""
 "Project-Id-Version: PCSX2 1.6.0\n"
 "Report-Msgid-Bugs-To: https://github.com/PCSX2/pcsx2/issues\n"
 "POT-Creation-Date: 2020-03-16 22:49+0100\n"
-"PO-Revision-Date: 2020-03-16 22:49+0100\n"
+"PO-Revision-Date: 2020-05-22 17:58+0200\n"
 "Last-Translator: StarFang208\n"
```
Comment (RedDevilus) [inline locales/it_IT/pcsx2_Main.po:9]: Should be SuperSamus

File: locales/it_IT/pcsx2_Main.po [hunk 2]
```diff
@@ -5,7 +5,7 @@ msgstr ""
 "Project-Id-Version: PCSX2 1.6.0\n"
```
Comment (RedDevilus) [inline locales/it_IT/pcsx2_Main.po:5]: PCSX2 1.7.0

File: locales/it_IT/pcsx2_Main.po [hunk 3]
```diff
@@ -158,40 +148,36 @@ msgstr "Non è stato possibile trovare il file del plugin %s configurato"
...
-msgstr ""
-"Il plugin %s configurato non è una libreria di collegamento dinamico valida"
+msgstr "Il plugin %s configurato non è una libreria di collegamento dinamico valida"
 
 #: pcsx2/PluginManager.cpp:976
 #, c-format
 msgid ""
-"The configured %s plugin is not a PCSX2 plugin, or is for an older "
-"unsupported version of PCSX2."
+"The configured %s plugin is not a PCSX2 plugin, or is for an older unsupported version of "
```
Comment (RedDevilus) [inline locales/it_IT/pcsx2_Main.po:156]: You have wrong line endings that mess with line endings

**Comment by RedDevilus** [comment]
https://u.pcloud.link/publink/show?code=XZvwnqkZTeC0BYAtTmzIAxiAXWcc0VLzjsSk

If you use Windows 10 then it needs different line endings.
Other method you need formatting set to unix.

**Comment by SuperSamus** [comment]
Following the instructions it changed some line endings anyway, but I hope it's still ok.

**Comment by lightningterror** [comment]
Is it ready?

**Comment by SuperSamus** [comment]
I thinks so?
I didn't check everything, I only corrected four things (three unnecessary periods and "Capttura">"Cattura"), there might more to correct.

**Pull request merged**